### ISSUE TITLE
Adding a warning when plugin not configured

### ISSUE THREAD

**Issue by palmiak** [itiden/statamic-buddy#6]
I started working on something like this
![image](https://user-images.githubusercontent.com/2342458/187429810-ac5a231e-72c3-4b76-8a70-0b9f653ff59a.png)

Until everything is configured user would get something like this - it is a rough draft, but I think it will be easier for users. If you think it's a good idea - I'll do a PR and work on this.

**Comment by danielbrodin** [comment]
Absolutely. I have been meaning to add something like this as well, so please continue.
